### PR TITLE
Agenda and waitlist

### PR DESCRIPTION
**Pull request by robdmoore**

### CODE REVIEW

File: pages/tickets.tsx [hunk 1]
```diff
@@ -67,6 +71,13 @@ class TicketPage extends React.Component<WithPageMetadataProps> {
       >
         <StyledContainer>
           <h1>Tickets</h1>
+          {this.props.pageMetadata.conference.IsSoldOut === SoldOutOptions.WaitList && (
+            <p className="alert alert-warning">
+              Tickets have sold out, but we are asking people to add themselves to the waitlist since it's likely we
+              will release more tickets. Tickets will be released to the waitlist on a first come fisrt served basis so
```
Comment (melissaettinger) [inline pages/tickets.tsx]: Typo in the second "first"

File: config/types.ts [hunk 2]
```diff
@@ -79,6 +79,12 @@ export interface Conference {
   Keynotes: Session[]
 }
 
+export enum SoldOutOptions {
```
Comment (melissaettinger) [inline config/types.ts]: TicketOptions or TicketStatus as a name for the enum could provide more clarity

File: pages/tickets.tsx (hunk 1 above)
Comment (robdmoore) [reply pages/tickets.tsx]: lol. This is what happens when I code when I'm tired and sick late at night. Thanks

File: config/types.ts (hunk 2 above)
Comment (robdmoore) [reply config/types.ts]: Totally agree, was being super lazy. Will fix 👍

**Pull request merged**